contrib/aws/aws-sdk-go/aws/aws_test.go

```diff
@@ -297,6 +303,194 @@ func TestWithErrorCheck(t *testing.T) {
 	})))
 }
 
+func TestExtraTagsForService(t *testing.T) {
```

Comment:
We probably want to test the case where the extra tags functions return a `nil` map. We want to make sure we catch any writes to the maps since `nil` map writes will panic.

Reply:
sounds good! added a subtest with EC2 (we don't set any extra tags for that one).